Homework9/Visitor.java

```diff
@@ -0,0 +1,20 @@
+package Homework9;
+
+import Homework2.*;
+
+public interface Visitor {
+    String visitTablet(Tablet computer);
+
+    String visitNotebook(Notebook computer);
+
+    String visitNetbook(Netbook computer);
+
+    String visitPersonalComputer(PersonalComputer computer);
+
+    String visitNettop(Nettop computer);
+
+    String visitMonoblock(Monoblock computer);
+
+    String visitServer(Server computer);
+}
+
```

Comment:
```suggestion
public interface Visitor<T> {
    T visit(Tablet computer);

    T visit(Notebook computer);

    T visit(Netbook computer);

    T visit(PersonalComputer computer);

    T visit(Nettop computer);

    T visit(Monoblock computer);

    T visit(Server computer);
}

```
это чтоб прям вообще красиво
так можно будет использовать посетитель для разных задач